### ISSUE TITLE
Suggestion: Repairman/Blacksmith Upgrad

### ISSUE THREAD

**Issue by High-hopes** [TangoTek/TekTopia-Community#312]
Suggestion: A way to fix villager tools. i.e. Repair miner's diamond pickaxe with another pick, or apply mending and be able to repair it that way. 

Method: 1.) Tools reaches x durability. > Villager places it back into community storage. >Blacksmith retrieves tool and combines with same to repair. Same way we do so. 
Method: 2.) ...back into community storage. > Blacksmith/Enchanter repairs mending tool.
Method: 3.) ... > Player retrieves tool and fixes it.

**Expected behavior**
Tool reaches a low level of durability. Villager goes and places it in the community storage. Villager/Player fixes said tool. Villager comes and retrieves fixed tool.

**Additional context**
Will add any edits required to clear up any confusion here.

**Comment by JohnyDL** [comment]
Related to #209

**Comment by Makion65** [comment]
I agree, and have been thinking the same thing. I have a lot of half-worn armor and tools/weapons in my chests, and don't really want to throw them out.

**Comment by JohnyDL** [comment]
ages and ages ago I asked for the guards to take off their armour and leave it in storage before bed so it could be repaired/upgraded

It's mean there'd also be a use for curse of binding -> don't want a villager to ever take off their armour then bind it to them ^_^ (I'd even like this to persist across job changes cause curse of binding feather falling would be a use on some of my narrow stairs)

and book/weapon combining would mean that gradually the armour and weapons would become the best possible

**Comment by Makion65** [comment]
I've tried combining 2 villager tools, and that works, but you get a player item then, not a villager one. That might be a bit hard to allow a player to end up crafting a villager item, but it could be a work around.
